data/ecosystems/m/moonriver.toml

```diff
@@ -54,7 +62,7 @@ sub_ecosystems = [
   "Zenlink",
 ]
 
-github_organizations = ["https://github.com/PureStake"]
+github_organizations = ["https://github.com/PureStake", "https://github.com/moonbeam-foundation", "https://github.com/Moonsong-Labs"]
```

Comment:
actually, should we remove these orgs from Moonriver and keep them in Moonbeam only instead? The taxonomy will rollup the relation between ecos anyways

Reply:
Totally agree! Done

Reply:
@eherrerosj Totally agree! Done